applications/luci-app-smartdns/root/usr/share/rpcd/acl.d/luci-app-smartdns.json

```diff
@@ -4,9 +4,8 @@
 		"read": {
 			"file": {
 				"/etc/smartdns/*": [ "read" ],
-				"/usr/sbin/iptables -t nat -nL PREROUTING": [ "exec" ],
-				"/usr/sbin/ip6tables -t nat -nL PREROUTING": [ "exec" ],
-				"/usr/sbin/smartdns": [ "exec" ]
+				"/usr/sbin/smartdns": [ "exec" ],
```

Comment:
No view code seems to invoke `/usr/sbin/smartdns` directly, so the permission should be dropped in order to not needlessly grant too many permissions.

Reply:
> I just noticed that this is a backport. Can you fix the mentioned issues in master so that I can simply cherry-pick them?

please review: 
https://github.com/openwrt/luci/pull/6022